guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/config/MavenBuildConfig.java

```diff
@@ -34,14 +34,14 @@
     */
     default List<String> getGoals() {
         final List<String> result = new ArrayList<>();
-        result.add( "package" );
+        result.add("package");
         return result;
     }
 
     default Properties getProperties() {
         final Properties result = new Properties();
-        result.setProperty("failIfNoTests", "false" );
+        result.setProperty("failIfNoTests",
+                           "false");
```

Comment:
It's not an error; it is intentional. The common opinion was that splitting parameters onto individual lines made stepping through method invocations easier when debugging. To be honest I'm completely fed up with talking about code formatting. We followed @hasys proposal; we maintained what @mdproctor wanted (parameter splitting) and we got the best "buy in" we've probably had..